### PR TITLE
Move Snap from OSS projects to Companies.

### PR DESCRIPTION
**Pull request by aiuto**
Move Snap from OSS projects to Companies.

### CODE REVIEW

File: users.md [hunk 1]
```diff
@@ -218,7 +218,7 @@ Ritual is a mobile pick up app, connecting restaurants with customers to offer a
 
 ### [Snap](https://www.snap.com/en-US/)
 
-Snap, the developper of Snapchat messaging app, has migrated from Buck to Bazel in 2020 ([source](https://twitter.com/wew/status/1326957862816509953)).
+Snap, the developper of Snapchat messaging app, has migrated from Buck to Bazel in 2020 ([source](https://twitter.com/wew/status/1326957862816509953)). Check out their [engineering blog](https://eng.snap.com/blog/) for more about their process.
```
Comment (floriographygoth) [inline users.md:221]: Editorial feedback. Ending statements with the link is easier for users. Rephrase to "For more details about their process, see their [engineering blog](https://eng.snap.com/blog/).

**Review by floriographygoth — APPROVED** [review]
Small editorial feedback, but otherwise LGTM.

File: users.md [hunk 2]
```diff
@@ -216,6 +216,10 @@ Redfin is a next-generation real estate brokerage with full-service local agents
 
 Ritual is a mobile pick up app, connecting restaurants with customers to offer a simple, time-saving tool to get the food and beverages they want, without the wait. Ritual uses Bazel for their backend services.
 
+### [Snap](https://www.snap.com/en-US/)
+
+Snap, the developper of Snapchat messaging app, has migrated from Buck to Bazel in 2020 ([source](https://twitter.com/wew/status/1326957862816509953)). Check out their [engineering blog](https://eng.snap.com/blog/) for more about their process.
```
Comment (nkoroste) [inline users.md:221]:
```suggestion
Snap, the developer of Snapchat messaging app, has migrated from Buck to Bazel in 2020. Check out their [engineering blog](https://eng.snap.com/blog/) for more about their process.
```

**Review by nkoroste — APPROVED** [review]

File: users.md (hunk 1 above)
Comment (aiuto) [reply users.md]: Done

**Pull request merged**